vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -4,34 +4,27 @@ sidebarDepth: 2
-6. [Deploy](#deploy)
-
-This document also describes the series of `ent ecr` commands that manage bundle interactions with the [Entando Component Repository](../../docs/compose/local-hub-overview.md) (ECR). These are applicable to both docker-based bundles and their git-based predecessors (< Entando 7.1). Finally, notable commands applicable to creating and deploying git-based bundles only are discussed.
+In addition, this document describes the series of `ent ecr` commands that manage bundle interactions with the [Entando Component Repository](../../docs/compose/local-hub-overview.md) (ECR). These are applicable to both docker-based bundles and their git-based predecessors (< Entando 7.1). Notable commands applicable to creating and deploying git-based bundles only are also discussed.
 
 ## Entando 7.1 Bundle Development
 
-Beginning with Entando 7.1, the `ent bundle` command and its convenience methods introduce a streamlined process to govern the structure, files and management of Entando bundles. The bundle development lifecycle is divided into 6 stages, each corresponding to a subcommand.
+Beginning with Entando 7.1, the `ent bundle` command and its convenience methods introduce a streamlined process to govern the structure, files and management of Entando bundles. The bundle development lifecycle is divided into 6 stages, each corresponding to a subcommand:
+
+- [Initialization](#initialization): `ent bundle init` initializes a new bundle project, either with the default files and folders or from an existing bundle in an Entando Hub. The bundle format relies on a single JSON descriptor as the project manifest. 
 
-The `init` subcommand initializes a new bundle project, either with the default files and folders or from an existing bundle in an Entando hub. The bundle format relies on a single JSON descriptor as the project manifest. 
+- [Build](#build): With the structure established, `ent bundle build` generates micro frontend (MFE) and microservice (MS) components. These are filtered by type and name and assigned version numbers. 
 
-With the structure established, the `build` subcommand executes processes appropriate to component type and stack.Micro frontend (MFE) and microservice (MS) components are generated in parallel. They are filtered by type and name and assigned version numbers. 
+- [Run](#run): `ent bundle run` and Keycloak integration enable components to be tested locally, external to an Entando cluster.   
 
-The `run` subcommand in conjuction with Keycloak integration enables components to be tested locally, external to an Entando cluster. Multiple components are run in parallel via commands dependent on type and stack, with a log file for each individual component generated in the .entando/logs directory.  
+- [Package](#package): `ent bundle pack` constructs the artifacts and Docker images for the bundle. A single image is created for the bundle and all micro frontends while each microservice is allocated its own image. 
```

Comment:
each microservice is allocated its own image. >> allocated seems inaccurate, it's generated or built

Reply:
"allocated" was an improvement on previous phrasing with "given." will reword.

Reply:
Let's try to match language we came up with elsewhere " `ent bundle pack` creates Docker images for the bundle and each of its microservices."

Reply:
Or alternatively, from below, "Generate the bundle artifacts, the bundle image and the microservice images"